test/integration/scheduler/util.go

```diff
@@ -150,33 +151,32 @@ func waitForNodeLabels(cs clientset.Interface, nodeName string, labels map[strin
 
 // initNode returns a node with the given resource list and images. If 'res' is nil, a predefined amount of
 // resource will be used.
-func initNode(name string, res *v1.ResourceList, images []v1.ContainerImage) *v1.Node {
-	// if resource is nil, we use a default amount of resources for the node.
+func initNode(name string, res *v1.ResourceList, images []v1.ContainerImage) *st.NodeWrapper {
 	if res == nil {
 		res = &v1.ResourceList{
 			v1.ResourcePods: *resource.NewQuantity(32, resource.DecimalSI),
 		}
 	}
 
-	n := &v1.Node{
-		ObjectMeta: metav1.ObjectMeta{Name: name},
-		Spec:       v1.NodeSpec{Unschedulable: false},
```

Comment:
@Huang-Wei do you know why were we setting this? AFAIK, there is no controller-manager in integration tests that would set this to true, or is there?

Reply:
`spec.Unschedulable` should be [deprecated](https://github.com/kubernetes/kubernetes/blob/cb37c0884664fd2f4a2800661cf92d7c32ee9143/pkg/scheduler/framework/plugins/nodeunschedulable/node_unschedulable.go#L60) IMO, so yes, we don't quite need to set it.

Reply:
Oh I didn't know, I will change it then, thanks!